torch/ao/sparsity/sparsifier/base_sparsifier.py

```diff
@@ -183,6 +163,7 @@ def _prepare(self, *args, **kwargs):
             module = config['module']
             param = config.get('parametrization', FakeSparsity)
             mask = config.get('mask', torch.ones(module.weight.shape))
+            self.state[config['fqn']]['mask'] = mask
```

Comment:
What is 'fqn' here?

Reply:
We use 'fqn' for a layer's "fully qualified name". Basically, it's the path that you can refer to using a series of `getattr` statements. I will document it in the docstrings and the design doc